content/reference/templates/index.md

```diff
@@ -1,60 +1,39 @@
-# Templates
+# index.liquid
 
-BookingSync themes are made up of Liquid template files, each serving their own unique purpose. For example, `search.liquid` is used to display multiple rentals, and `rental.liquid` is used to show the details of a single rental.
+The `index.liquid` template renders the homepage of the website. The template is loaded when you go to the root ("/") of the website.
 
-## Theme Structure
+## Components
 
-A BookingSync theme comprises the following directories:
+### Home slider
 
-* images
-* javascripts
-* layouts
-* snippets
-* stylesheets
-* templates
-
-### Images directory
-
-The `images` directory contains all the images used in the theme, including theme images and vendors images.
-
-### Javascripts directory
-
-The `javascripts` directory contains all the javascripts files used in the theme.
-
-You can write javascript using `javascript` or `coffeescript` languages.
+~~~ liquid
+{% include 'snippets/home_slider' %}
+~~~
 
-### Layouts directory
+This component renders a photos slider using promoted locations main photo.
 
-The `layouts` directory contains theme layout templates. All Liquid templates inside the templates folder are rendered inside the `theme.liquid` file.
+### Search bar
 
-### Snippets directory
+~~~ liquid
+{% include 'snippets/search_bar' %}
+~~~
 
-The `snippets` directory contains all the theme's Liquid snippet files, which are bits of code that can be referenced in other templates of a theme.
+This component renders a search form. According to your settings, the form will display these inputs:
 
-Use the `include` tag to load a snippet into your theme.
+* Destination dropdown
+* Checkin datepicker
+* Checkout datepicket
+* Number of people dropdown
+* Amenities checkboxes
 
-#### Example:
+### Promoted categories
 
 ~~~ liquid
-{% include 'snippets/home_slider' %}
+{% include 'snippets/home_categories' %}
 ~~~
 
-### Stylesheets directory
-
-The `stylesheets` directory contains all the CSS files used in the theme.
-
-You can write CSS using `css`, `scss` or `sass` languages.
-
-### Templates directory
+This component renders a list of promoted categories.
 
-The `templates` directory contains all other Liquid templates:
+## Parameters
 
-* `404.liquid`
-* `category.liquid`
-* `contact.liquid`
-* `destination.liquid`
-* `index.liquid`
-* `info_window.liquid`
-* `page.liquid`
-* `rental.liquid`
-* `search.liquid`
+* &debug=1 : Switch to debug mode
```

Comment:
Should be a table I think, with column A the name of the param, column B with the type, column C the description and default value. cf. API Doc http://developers.bookingsync.com/reference/endpoints/accounts/
No need of the Read/Write here, not applicable.